third_party/libunwindstack/Unwinder.cpp

```diff
@@ -211,6 +239,16 @@ void Unwinder::Unwind(const std::vector<std::string>* initial_map_names_to_skip,
       // Once a frame is added, stop skipping frames.
       initial_map_names_to_skip = nullptr;
     }
+    if (map_info != nullptr && ShouldStop(functions_to_stop, map_info->name(), rel_pc)) {
```

Comment:
Is there a _simple_ way to add a _simple_ unit test for this? Is there an existing test that we can copy and modify by passing the new argument?

Reply:
Unfortunately not... There are actually not even unit tests for the existing "should stop" case...

Reply:
As discussed, there are actually tests. I'll add a test asap.